pkg/plugins/common/kustomize/v2-alpha/scaffolds/api.go

```diff
@@ -95,6 +82,13 @@ func (s *apiScaffolder) Scaffold() error {
 		); err != nil {
 			return fmt.Errorf("error scaffolding kustomize API manifests: %v", err)
 		}
+
+		// If the gvk is non-empty
+		if s.resource.Group != "" || s.resource.Version != "" || s.resource.Kind != "" {
+			if err := scaffold.Execute(&samples.Kustomization{}); err != nil {
+				return fmt.Errorf("error scaffolding manifests: %v", err)
+			}
+		}
```

Comment:
nits: we may remove the method `generateManifestsPath` as it is unused.

Reply:
good catcher :-)